# Generated for NaoQi method ALLogger.warn
# Log a warning.

# Name of the module.
string moduleName
# Log Message.
string message
---


